### PR TITLE
#2494 AC Emulation : use the proper p0 and start value for delta theta filtered

### PR DESCRIPTION
**Pull request by cossartque**
**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non regression tests were added (new model, new feature and bug fix)
- [ ] main documentation was updated (update of input/output file, 3rd party, model, repository organization, solver)
- [ ] example documentations were updated (new example in examples folder)
- [ ] if this PR modifies the parameters or inputs/outputs of a model/solver: the corresponding xsl was added in util/xsl and platform DB were updated
- [ ] if this PR modifies a dictionary: the corresponding french dictionary was updated
- [ ] if this commit targets a release branch: the corresponding milestone was added in the ticket and in this PR

### CODE REVIEW

File: dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNHvdcLineInterfaceIIDM.cpp [hunk 1]
```diff
@@ -155,8 +155,13 @@ void HvdcLineInterfaceIIDM::importStaticParameters() {
   staticParameters_.clear();
 
   bool isACEmulationEnabled = hvdcActivePowerControl_ && hvdcActivePowerControl_->isEnabled() && hvdcActivePowerControl_->isEnabled().get();
+  double p0ACEmulation = !isACEmulationEnabled ? 0 : hvdcActivePowerControl_->getP0().get();
+  double p0ACEmulationPu = (hvdcLineIIDM_.getConvertersMode() == powsybl::iidm::HvdcLine::ConvertersMode::SIDE_1_RECTIFIER_SIDE_2_INVERTER) ?
+                           p0ACEmulation / SNREF : - p0ACEmulation / SNREF;
   staticParameters_.insert(std::make_pair("isACEmulationEnabled", StaticParameter("isACEmulationEnabled",
                              StaticParameter::BOOL).setValue(isACEmulationEnabled)));
+  staticParameters_.insert(std::make_pair("p0ACEmulationPu", StaticParameter("p0ACEmulationPu",
+                             StaticParameter::DOUBLE).setValue(p0ACEmulationPu)));
```
Comment (adriguir) [inline dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNHvdcLineInterfaceIIDM.cpp:164]: I don't really like having the sign change hidden into the pu transformation. I would suggest an alternative implementation, that I find clearer:

double p0ACEmulationPu = 0;
if (isACEmulationEnabled) {
  double p0ACEmulation = (hvdcLineIIDM_.getConvertersMode() == ) .. hdvcActivePowerControl ... : - hvdcActivePowerControl...
  p0ACEmulationEnabledPu = p0ACEmulation / SNREF;
}

**Review by adriguir — CHANGES_REQUESTED** [review]
Also add a test on the p0ACEmulationAC calculation into the unit test of PowSyBlIIDM (missing in the current state of the dev)

File: dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNHvdcLineInterfaceIIDM.cpp (hunk 1 above)
Comment (cossartque) [reply dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNHvdcLineInterfaceIIDM.cpp:164]: Ok, thanks

**Comment by cossartque** [comment]
> > Also add a test on the p0ACEmulationAC calculation into the unit test of PowSyBlIIDM (missing in the current state of the dev)
> 
> I'll add the test in dynawo-rte as the extension is not accessible here in dynawo (it seems)

In fact, everything is already tested in dynawo-rte unit tests + the nrt I created

**Review by adriguir — APPROVED** [review]

**Pull request merged**